### PR TITLE
OFS-279: added 'relevance' and 'condition'

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openimis.atlassian.net/browse/OFS-279
- solved point a) and b)

### CODE REVIEW

File: calculation/config.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
                     "update": "151203",
                     "replace": "151206",
                 },
+                'relevance': "True",
+                'condition': ".$_>1",
```
Comment (delcroip) [inline calculation/config.py:19]: is the "." on purpose (like to test this way of writing) ? (you put <100 in other)

**Review by delcroip — APPROVED** [review]

File: calculation/config.py (hunk 1 above)
Comment (rstencel) [reply calculation/config.py:19]: @delcroip @sniedzielski can we change `$_` for `INPUT`?

Comment (sniedzielski) [reply calculation/config.py:19]: Screenshot from 2021-04-15 11-32-27 it is on wiki page that dot. I can quickly fix this

Comment (sniedzielski) [reply calculation/config.py]: @rstencel @delcroip No problem for me to change this $_ into INPUT

**Comment by sniedzielski** [comment]
@delcroip I've just changed for @rstencel the "condition" values. You can re-check

File: calculation/config.py (hunk 1 above)
Comment (delcroip) [reply calculation/config.py]: No issue for me either

**Review by delcroip — APPROVED** [review]

**Pull request merged**